### DISCUSSION TITLE
Developer Documentation

### DISCUSSION THREAD

**Discussion by joseph254ke** [IndiePython/nodezator#98] [Q&A]
Will there be a developer documentation that shows how the whole codebase works

**Comment by KennedyRichard** [comment]
Yes. The document will describe each service and how the app was designed in the detail. I already started writing this document, but it is far from complete: https://github.com/IndiePython/nodezator-sdd

It's been 05 months since I last touched the document but that's because I decided to prioritize the next Nodezator release at the time. I intend to fully finish it in the next couple months or so. In fact, I do need to do so in order to be able to finish the scheduled development work for Nodezator, as explained here: #97
